lib/hpke/src/userinfo_vc.cpp

```diff
@@ -222,7 +222,7 @@ struct UserInfoVC::ParsedCredential
       epoch_time(payload.at("nbf").get<int64_t>()),
       epoch_time(payload.at("exp").get<int64_t>()),
 
-      vc.at("credentialSubject"),
+      UserInfoClaims::from_json(vc.at("credentialSubject").dump()),
```

Comment:
If you define `to_json / from_json` for `UserInfoClaims, you should be able to just pass `vc.at("credentialSubject")`.

https://github.com/nlohmann/json#basic-usage

Reply:
that would require the end-user to using nlohmann/json.